lib/linked_in/api.rb

```diff
@@ -1,13 +1,17 @@
+require 'date'
+
 module LinkedIn
   class API
 
-    attr_accessor :access_token
+    attr_accessor :access_token, :linkedin_api_version
 
-    def initialize(access_token=nil)
+    def initialize(access_token=nil, linkedin_api_version=nil)
```

Comment:
Can we rename `linkedin_api_version` to `api_version`? as it conflicts with attr_accessor?
```suggestion
    def initialize(access_token=nil, api_version=nil)
```